apps/code-examples/src/app/code-examples/popovers/dropdown/basic/demo.component.html

```diff
@@ -12,3 +12,17 @@
     </sky-dropdown-item>
   </sky-dropdown-menu>
 </sky-dropdown>
+<br />
+<sky-dropdown buttonType="context-menu" label="Code example context menu">
+  <sky-dropdown-menu>
+    <sky-dropdown-item *ngFor="let item of items">
+      <button
+        type="button"
```

Comment:
```suggestion
        type="button"
        class="sky-btn sky-btn-default"
```
I assume this button should demonstrate using our button classes? ... And if so, I assume we also need to add it to the existing button as well.

Reply:
It actually shouldn't! It is the item within the dropdown, so it isn't styled the way our standalone buttons are